src/Blog/View/posts.twig

```diff
@@ -1,25 +1,71 @@
 {% set posts = [
     {
         'id': '1',
-        'title': 'Lorem ipsum',
+        'title': 'Jak efektywnie uczyć się programowania?',
         'preamble': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
         'url':'/img/slider1.svg',
         'date':'10-02-2020'
     }
-] %}
+	] %}
 {% include 'Common/View/header.twig' %}
 {% block body %}
-	<body class="top-wave">
+	<body class="top-wave" style="font-family:Lato;">
```

Comment:
That is correct, that font-family property should be in defined in root element, like body. In your solution, you use inline way to adding style. It is the most specyfic way to set style for element. 
Font Lato is already set in _global.css file. So, if you want, you can delete it, and ride of repetition. I think it is better practice to add style in css file. Place for fonts is in `body` type selector.

Reply:
it doesn't work without Lato in body selector

Reply:
hymn, so maybe I was wrong. I see that font-family property is added to `html` element, not to `body`. So, my by tray this way? or leave it. Both solutions, your and with `html` will work